napalm/__init__.py

```diff
@@ -26,6 +27,7 @@ def get_network_driver(vendor):
         'JUNOS': JunOSDriver,
         'JUNIPER': JunOSDriver,
         'FORTIOS': FortiOSDriver,
+#        'IBM': IBMDriver,
```

Comment:
@fooelisa thanks, I will try to use this guide :) 
@dbarrosop  Sorry for misunderstanding, module has been tested on 7.11.2 version and it should work on all 7.x.x version. There is newest branch with number 8.x but it was prepared only for Lenovo branded device. I'm sure that all from 7.x.x will work but I have to confirm what about 8.x :)